testsuite/forge-cli/src/main.rs

```diff
@@ -437,7 +437,10 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 10000,
                 false,
                 Some(Duration::from_secs(240)),
-            )),
+            ))
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 30.into();
```

Comment:
let's do 20, so that we actually make sure it happens? or increase the time below to 40s?

Reply:
It's guaranteed to happen because the run-time of the test is several minutes - we emit transactions for 30s multiple times in this test.